### PR TITLE
chore: log missing processors as errors

### PR DESCRIPTION
**Pull request by tartansandal**

### CODE REVIEW

**Review by lukemerrett — APPROVED** [review]

File: sqs_workers/queue.py
Comment (Schnouki) [inline sqs_workers/queue.py:588]: nit: there should be a space at the end of the first string

Comment (tartansandal) [reply sqs_workers/queue.py:588]: Ah good catch 🎣. Next time I'll wait for your review before merging🤦.

Comment (Schnouki) [reply sqs_workers/queue.py:588]: No worries 😊 And no need to wait either if you already have a ✅, that's mostly me being a bit of a perfectionist 😅

**Pull request merged**
